src/sharing/index.jsx

```diff
@@ -1,9 +1,11 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
+import flow from 'lodash/flow'
 import createReactContext from 'create-react-context'
-import { Query } from 'cozy-client'
+import { queryConnect, withClient } from 'cozy-client/dist/hoc'
```

Comment:
Why not just: 

import { queryConnect, withClient } from 'cozy-client' ?

Reply:
it breaks `SharingsContainer` test because it mocks `cozy-client` and `lodash.flow` wants that the result of `queryConnect(...)` is a function. As I said this morning, I think I can just move `SharedDocuments` outside of the file and it should work.